### ISSUE TITLE
Enter a title here

### ISSUE THREAD

**Issue by cas-1092** [RPCS3/rpcs3#15762]
### Quick summary

The game's motion blur causes a strange ghosting effect to occur.

### Details

Real PS3: https://www.youtube.com/watch?v=CmgG-IDz7yw (Timestamp 30:21 for comparison)

RPCS3: https://www.youtube.com/watch?v=DEM1M4a19gA

The effect is very visible whenever nitrous is used,
Settings like WCB, RDB, WDB, RCB do not help either. 

### Attach a log file

Log: [Need for Speed ProStreet [BLUS30066].log.gz](https://github.com/user-attachments/files/16048946/Need.for.Speed.ProStreet.BLUS30066.log.gz)

### Attach capture files for visual issues

RSX Capture:  [BLUS30066_20240701024835_capture.rrc.gz](https://github.com/user-attachments/files/16048973/BLUS30066_20240701024835_capture.rrc.gz)
Dont know if I did this right.

### System configuration

Windows 11 23H2 | Intel Core i7-13700F | 31.85 GiB RAM | NVIDIA GeForce RTX 4060 Ti (555.85)

### Other details

Only setting that was changed is PPU decoder to Interpeter (Static) as it fixes crashing, but even with all default settings it still occurs.

**Comment by Megamouse** [comment]
Your dump is broken or you don't have the proper license.

**Comment by cas-1092** [comment]
No, my dump is not broken as I checked with PS3isorebuilder and it returned all green.
![image](https://github.com/RPCS3/rpcs3/assets/137835937/7e117539-8301-49e6-9257-c39dc9d22990)


If by license you mean DLC, I applied none, only two updates.

**Comment by digant73** [comment]
@cas-1092 is that still an issue?

**Comment by cas-1092** [comment]
<img width="1881" height="1040" alt="Image" src="https://github.com/user-attachments/assets/bc7b9fcd-c4df-462c-87cc-753796ca53c1" /> 
Yes, this is still an issue.

**Comment by digant73** [comment]
@cas-1092 please re-test

**Comment by cas-1092** [comment]
<img width="1919" height="1079" alt="Image" src="https://github.com/user-attachments/assets/072b1bc7-3f5b-43f8-b82a-d1cbbf51af81" />
Ghosting is still present.

**Comment by sergus2008** [comment]
<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/cb94823a-36e5-4657-b9a9-482a6f725bfe" />

Ghosting is still present on latest master with PR №19383

**Comment by Asinin3** [comment]
Dont tag pull-requests if they aren't related. Just give version number (your screenshot already has it). Only tag PRs if they caused a bug or you need to mention them.
